### ISSUE TITLE
Add isHoliday() method - Enhancement

### ISSUE THREAD

**Issue by hmrneves1** [briannesbitt/Carbon#2206]
Hello, I'm using this package on a course that I'm taking and I think the packages lacks of holiday checking. I have researched this subject and I saw that this feature was rejected many times.

Obviously that the package maintainer won't know every holiday in every country.

I though that we should implement this like Languages. Having an "Holiday" folder with files per country, like "Holidays/pt.php", with an array of dates. We could populate this very quickly with community support.

I created a fork but don't really know how to implement lol

Basically:

```php
$date = Carbon::parse('2020-09-30');

// argument is the country code, in this case, portuguese
if ($date->isHoliday('pt') true
else false;
```

Just a suggestion :)

Best Regards,
Helder

**Comment by kylekatarnls** [comment]
Hello,

The need is fulfilled by [cmixin/business-day](https://github.com/kylekatarnls/business-day) and mentioned in the Carbon documentation, it supports many countries and custom lists.

Thanks,

**Comment by hmrneves1** [comment]
Hi, totally missed this :O thanks and I'm sorry!

Regards,
Helder
